### ISSUE TITLE
Broke Error Handling

### ISSUE THREAD

**Issue by antl3x** [kamilogorek/pglite-server#1]
Hey, thanks for the great lib! 

I was playing with and I think that error handling is not working properly? I ran a psql invalid command and the node process got an exit 1 and nothing was sent back to psql.

``` psql
postgres=> INVALID COMMAND;
server closed the connection unexpectedly
        This probably means the server terminated abnormally
        before or while processing the request.
The connection to the server was lost. Attempting reset: Failed.
The connection to the server was lost. Attempting reset: Failed.
!?> 
```

``` node

error: syntax error at or near "INVALID"
    at Ke.parseErrorMessage (/node_modules/@electric-sql/pglite/dist/chunk-OVRU7FYL.js:1:12967)
    at Ke.handlePacket (/node_modules/@electric-sql/pglite/dist/chunk-OVRU7FYL.js:1:9640)
    at Ke.parse (/node_modules/@electric-sql/pglite/dist/chunk-OVRU7FYL.js:1:8285)
    at <anonymous> (/node_modules/@electric-sql/pglite/dist/chunk-OVRU7FYL.js:1:23863)
    at Array.forEach (<anonymous>)
    at <anonymous> (/node_modules/@electric-sql/pglite/dist/chunk-OVRU7FYL.js:1:23841)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  length: 95,
  severity: 'ERROR',
  code: '42601',
  detail: undefined,
  hint: undefined,
  position: '1',
  internalPosition: undefined,
  internalQuery: undefined,
  where: undefined,
  schema: undefined,
  table: undefined,
  column: undefined,
  dataType: undefined,
  constraint: undefined,
  file: 'scan.l',
  line: '1181',
  routine: 'scanner_yyerror'
}

Node.js v20.10.0
```

**Comment by antl3x** [comment]
Probably we need a `try .. catch` here:

https://github.com/kamilogorek/pglite-server/blob/5e5e8d3f3fdf5c43694a793eca00bfdae5d76400/src/responses.ts#L53-L56

**Comment by kamilogorek** [comment]
Fixed in `0.1.2`, thanks for the report :)
